features/admin.console-settings.v1/components/console-roles/console-roles-list-layout.tsx

```diff
@@ -195,20 +195,23 @@ const ConsoleRolesListLayout: FunctionComponent<ConsoleRolesListLayoutPropsInter
             onItemsPerPageDropdownChange={ handleItemsPerPageDropdownChange }
             onPageChange={ handlePaginationChange }
             showTopActionPanel={ (rolesList?.totalResults > 0 || searchQuery?.length !== 0) }
-            topActionPanelExtension={ !isSubOrganization() && (
-                <Show when={ featureConfig?.userRoles?.scopes?.create }>
-                    <PrimaryButton
-                        data-componentid={ `${componentId}-add-button` }
-                        onClick={ () => onRoleCreate() }
-                    >
-                        <Icon
-                            data-componentid={ `${componentId}-add-button-icon` }
-                            name="add"
-                        />
-                        { t("roles:list.buttons.addButton", { type: "Role" }) }
-                    </PrimaryButton>
-                </Show>
-            ) }
+            topActionPanelExtension={
+                !isSubOrganization() &&
+                // isConsoleRolesEditable &&
```

Comment:
Remove comment if not needed

Reply:
Fixed with [d587fe7](https://github.com/wso2/identity-apps/pull/6938/commits/d587fe7b7259c4a7ddb0ce2652dc1c38c1278380)